development.ini

```diff
@@ -5,7 +5,10 @@ unicore.content_repos =
     unicore_frontend_straighttalk_za
     unicore_frontend_ureport_za
     unicore_frontend_mariestopes_za
-    unicore_frontend_ffl_za
+
+iogt.content_section_url_overrides =
+    ffl = http://za.ffl.qa-hub.unicore.io/
```

Comment:
This is going to be a problem. We'll need to link to to `/section/ffl/` rather than outside of the app.